### PR TITLE
Bugfix/split on undefined

### LINKED ISSUE #3: .split on undefined causes a crash
It can happen that the trigger ends up in an error as below:

TypeError: Cannot read properties of undefined (reading 'split')
    at Function.formatLatestChanges (/directus/extensions/hooks/version-control-changelog/index.js:8:8352)
    at Bi.triggerProcessor (/directus/extensions/hooks/version-control-changelog/index.js:8:19511)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

The reason still needs to be investigated, but the guess is that the system cannot handle formatting of a NULL value in the previous change log data yet (as this appears on a system that never had any content in that field before)

### PR DESCRIPTION
**Pull request by ManuelFeller**
This fixes #3 and also adds an improvement to the logger's prefix on non colored consoles

### CODE REVIEW

File: src/core/ChangelogWriter.ts [hunk 1]
```diff
@@ -99,11 +99,18 @@ export default class ChangelogWriter {
 
 		const changeUserId = this.extractUserId(userDetails[0]);
 		const date = new Date().toLocaleString('en-GB', { hour12: false, timeZone: 'UTC', timeZoneName: 'short' });
-		const newData = ChangelogFormatter.formatLatestChanges(event.payload.changes,`## ${date} by ${changeUserId}`);
-		const commitMsg = `Content update from Directus CMS by ${changeUserId}`;
-
-		// do the actual execution
-		this.vcsAbstraction.addToChangelogFile(newData, commitMsg);
+		const changeContent = event.payload[this.extensionConfig.fieldName];
+		if (event.payload[this.extensionConfig.fieldName] !== undefined) {
```
Comment (gerardcl) [inline src/core/ChangelogWriter.ts]:
```suggestion
		if (changeContent !== undefined) {
```

Comment (ManuelFeller) [reply src/core/ChangelogWriter.ts]: Ohh, yes (shame on me)

File: src/core/ExtensionLogger.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ export default class ExtensionLogger {
 		// using "name" to have it before the message itself based on the used formatter
 		// --> see https://github.com/directus/directus/blob/ee7e678f24fd85846798bc3cc1af3103e728ebb7/api/src/logger.ts#L18
 		// and https://www.npmjs.com/package/pino-colada
-		this.loggerReference = directusLogger.child({name: 'Version Control Changelog Extension'});
+		this.loggerReference = directusLogger.child({name: 'Version Control Changelog Extension >'});
```
Comment (gerardcl) [inline src/core/ExtensionLogger.ts:21]: is this `>` symbol expected?

Comment (ManuelFeller) [reply src/core/ExtensionLogger.ts:21]: Yes, to make it more obvious where the prefix ends when you do not have colors in the console

**Review by gerardcl — APPROVED** [review]
LGTM! 🚀

**Review by cschweikert — APPROVED** [review]
@gerardcl you was always one second faster with your comments 😄
LGTM2

**Pull request merged**